jwst/pipeline/calwebb_coron3.py

```diff
@@ -16,10 +16,6 @@
 
 __version__ = "0.7.1"
 
-# Define logging
-import logging
-log = logging.getLogger()
-log.setLevel(logging.DEBUG)
```

Comment:
I wonder what the default Level is set to in the stpipe Step class. We always explicitly set it to DEBUG in the individual step and pipeline classes to make sure that the logger will pass DEBUG level messages when requested to do so via the verbose argument. If the default in the stpipe Step class is INFO, for example, then the pipelines and steps may not pass DEBUG level messages. The setLevel seems to act as an "upper limit" as to what can get passed.

Reply:
The default setup in `step.py` allows passing of DEBUG level messages.  I've had `Image3Pipeline` running like this for a long time.  The steps within that pipeline have the same setup.  The logging setup is inherited from the parent class.